### PR TITLE
Fix use of multiple configuration sources for liquid configuration

### PR DESCRIPTION
**Pull request by aledaccas**
Restructuring of project folders and correction for the use of multiple configuration sources (Environment Variables, secrets, etc.)

resolves https://github.com/Avanade/Liquid-Application-Framework/issues/35

### CODE REVIEW

File: src/Liquid.Core/Configuration/LightFileConfigurationException.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ namespace Liquid.Core.Configuration
     /// Occurs when it is not possible to read the appsettings file.
     /// </summary>
     /// <seealso cref="Liquid.Core.Exceptions.LightException" />
+    [Serializable]
     [ExcludeFromCodeCoverage]
     public class LightFileConfigurationException : LightException
```
Comment (lucianareginalino-avanade) [inline src/Liquid.Core/Configuration/LightFileConfigurationException.cs:13]: missing ISerializable constructor implementations.

File: src/Liquid.Core/Exceptions/LightCustomException.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@ namespace Liquid.Core.Exceptions
     /// Class responsible for custom exception codes handling.
     /// </summary>
     /// <seealso cref="LightException" />
+    [Serializable]
     [ExcludeFromCodeCoverage]
     public class LightCustomException : LightException
```
Comment (lucianareginalino-avanade) [inline src/Liquid.Core/Exceptions/LightCustomException.cs:12]: missing ISerializable constructor implementations.

File: src/Liquid.Core/Localization/LocalizationException.cs [hunk 3]
```diff
@@ -8,6 +8,7 @@ namespace Liquid.Core.Localization
     /// Occurs when it's not possible to read a resource.
     /// </summary>
     /// <seealso cref="System.Exception" />
+    [Serializable]
     [ExcludeFromCodeCoverage]
     public class LocalizationException : LightException
```
Comment (lucianareginalino-avanade) [inline src/Liquid.Core/Localization/LocalizationException.cs:13]: missing ISerializable constructors implementation.

File: src/Liquid.Core/Localization/LocalizationReaderException.cs [hunk 4]
```diff
@@ -8,6 +8,7 @@ namespace Liquid.Core.Localization
     /// Occurs when it's not possible to read the resources collection from data source.
     /// </summary>
     /// <seealso cref="System.Exception" />
+    [Serializable]
     [ExcludeFromCodeCoverage]
     public class LocalizationReaderException : LightException
```
Comment (lucianareginalino-avanade) [inline src/Liquid.Core/Localization/LocalizationReaderException.cs:13]: missing ISerializable constructor implementations.

File: src/Liquid.Core/Configuration/LightFileConfigurationException.cs (hunk 1 above)
Comment (aledaccas) [reply src/Liquid.Core/Configuration/LightFileConfigurationException.cs:14]: Resolved

File: src/Liquid.Core/Exceptions/LightCustomException.cs (hunk 2 above)
Comment (aledaccas) [reply src/Liquid.Core/Exceptions/LightCustomException.cs:13]: Resolved

File: src/Liquid.Core/Localization/LocalizationException.cs (hunk 3 above)
Comment (aledaccas) [reply src/Liquid.Core/Localization/LocalizationException.cs:14]: Resolved

File: src/Liquid.Core/Localization/LocalizationReaderException.cs (hunk 4 above)
Comment (aledaccas) [reply src/Liquid.Core/Localization/LocalizationReaderException.cs:14]: Resolved

**Review by lucianareginalino-avanade — APPROVED** [review]

**Pull request merged**